pages/index.js

```diff
@@ -164,6 +192,48 @@ class Home extends React.Component {
               </Link>
             </div>
           </section>
+          <Divider hidden />
+          <section className="discord">
+            <div className="container">
+              <h3 className="taglines">
+                Join our Discord server, and say Hello World!
+              </h3>
+              <Button.Group basic>
+                <Button
+                  content="Join Chat"
+                  icon="discussions"
+                  labelPosition="left"
+                  onClick={() =>
+                    window.open('https://discord.gg/dVnQ2Gf', '_blank')}
+                />
+              </Button.Group>
+            </div>
+          </section>
+          <section className="update">
+            <div className="container update_container">
+              <h3 className="taglines">
+                We are constanly updating our platform.<br />If you would like
+                to stay informed about our updates, drop you email.
+              </h3>
+              <div className="update_content">
+                {this.state.subscriberEmailPosted ? (
+                  <h2>Thank you, we will keep you posted</h2>
+                ) : (
+                  <Form onSubmit={this.handleSubmit}>
+                    <Form.Group>
+                      <Form.Input
+                        placeholder="Enter email address"
+                        name="email"
+                        value={this.state.subscribersEmail}
+                        onChange={this.handleChange}
+                      />
+                      <Form.Button color="pink" content="Subscribe" />
```

Comment:
Can you add loading state to button when email is posting to backend
Ref: https://react.semantic-ui.com/elements/button#button-example-loading